src/main/java/kappzzang/jeongsan/global/common/enumeration/ErrorType.java

```diff
@@ -0,0 +1,20 @@
+package kappzzang.jeongsan.global.common.enumeration;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public enum ErrorType {
+
+    USER_NOT_FOUND(HttpStatus.NOT_FOUND, "000", "사용자를 찾을 수 없습니다.");
+
+    private final HttpStatus httpStatus;
+    private final String errorCode;
+    private final String message;
+
+    ErrorType(HttpStatus httpStatus, String errorCode, String message) {
+        this.httpStatus = httpStatus;
```

Comment:
P2: 부모 클래스의 메서드 (`isSameCodeAs`, `valueOf`)를 사용하게 될 지는 모르겠으나 확장성을 고려했을 때 선언 타입을 부모 클래스인 `HttpStatusCode`를 사용하는 게 어떨까요?

```suggestion
    USER_NOT_FOUND(HttpStatus.NOT_FOUND, "000", "사용자를 찾을 수 없습니다.");

    private final HttpStatusCode httpStatusCode;
    private final String errorCode;
    private final String message;

    ErrorType(HttpStatusCode httpStatusCode, String errorCode, String message) {
        this.httpStatusCode = httpStatusCode;
```